repos.d/deb/deepin.yaml

```diff
@@ -2,11 +2,9 @@
 # Deepin Linux
 ###########################################################################
 
-# Though we reserve place for deepin_{testing,stable} here, unstable
-# is currently the only repo (not taking precise/trusty into account)
-- name: deepin_unstable
+- name: deepin_20
```

Comment:
Please don't rename the repository as it's the same as removing and adding a new repository, loosing all history. You can add `sortname: deepin_20` here so it's ordered correctly against newer versions (#1357).